### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by celsom3**
NOTE: all links and image references now have to have this (`{% if site.github.is_project_page %}/guidelines{% endif %}`) as a prefix.

Not sure if that is worth it to be able to run this locally, but it does make running it locally less of a hassle.

### CODE REVIEW

**Comment by swilliams** [comment]
Does this load the guidelines jekyll site into a docker container?

**Comment by swilliams** [comment]
Ok, yes it does. :-)

File: docs/Code.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
+layout: default
 title: "Tallwave Guidelines — Code"
 ---
 
-[&larr; Home](/guidelines)
+[&larr; Home]({% if site.github.is_project_page %}/guidelines{% endif %}/)
```
Comment (swilliams) [inline docs/Code.md]: Is it possible to add variables to `site` to make lines like this a little easier?

Like:

``` ruby
{%
  site.root = if site.github.is_project_page then '/guidelines' else ''
%}

```

Then you could have links like `[whatever]({% site.root %}/thing)`.

(my ruby is super rust too)

Comment (celsom3) [inline docs/Code.md]: That's a good idea. I'll try that out.

**Comment by celsom3** [comment]
@swilliams I tried something similar. I couldn't run that if statement inside the _config.yml, so instead I specified a different --config file.

**Pull request merged**